chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.ts

```diff
@@ -12,7 +11,7 @@ import index from "@angular/cli/lib/cli";
 export class CustomerSummaryComponent implements OnInit {
   public customers: CustomerInfo [] = [];
 
-  constructor() {
+  constructor(public generateCustomerService: GenerateCustomerService) {
```

Comment:
Rename `generateCustomerService` property to be just `customerService`.

Reply:
Replaced generateCustomerService with customerService.